core/views.py

```diff
@@ -104,7 +104,25 @@ class ReportsView(LoginRequiredMixin, TemplateView):
 
 @login_required
 def entries_csv_export(request):
```

Comment:
Need to rename to reports_export and change URL in urls.py too, make sure tests are updated as well.